onfleet/exceptions.py

```diff
@@ -4,3 +4,7 @@ class OnfleetException(Exception):
 
 class OnfleetDuplicateKeyException(OnfleetException):
     pass
+
+
+class OnfleetGeocodingException(OnfleetException):
+    pass
```

Comment:
Does it make sense to map out all the possible onfleet errors in one fell swoop? I'd be into the idea, instead of adding them one by one as we meet them. 

I found this document linked from their docs — https://gist.github.com/euskode/8a34b2152cb35ddeb659 — which appears to describe the possible errors. But it hasn't been updated in a year and doesn't have any geocoding error in it. Maybe we could ask Onfleet directly for a list of possible errors?